main.rb

```diff
@@ -1,100 +1,138 @@
 require_relative('./app')
```

Comment:
Great job team 🤝

- [ ] Kindly ensure your `main. rb` file is only to start your code and ask the user for the initial choice of options (or to quit). I can see you implemented the method logic in your `main. rb` file. I would suggest the logic is implemented in separate files and then imported into the `main. rb` file. This will make your project conform to the Single Responsibility Rule.